langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -124,6 +229,27 @@ public isolated function reduce(Type[] arr, @isolatedParam function(Type1 accum,
 # The parameter `func` is called for each member of parameter `arr` in order unless and until a call returns true.
 # When the array is empty, returns false.
 #
+# ```ballerina
+# int[] numbers = [1, 2, 3, 5];
+# boolean hasEvenNumber = numbers.some((number) => number % 2 == 0);
+#
```

Comment:
Again, would it be better if a comment is added here to distinguish between the two examples?